app/views/lessons/show.html.erb

```diff
@@ -7,8 +7,29 @@
 
     <main class="grid grid-cols-12 gap-x-6" data-controller="lesson-toc" data-lesson-toc-item-classes-value="no-underline hover:text-gray-800 text-sm dark:hover:text-gray-300">
       <article class="col-span-full xl:col-span-7 xl:col-start-2">
-        <%= render ContentContainerComponent.new(classes: 'xl:mx-0', data_attributes: { lesson_toc_target: 'lessonContent' }) do %>
-          <%= @lesson.body.html_safe %>
+        <%= render ContentContainerComponent.new(classes: 'xl:mx-0', data_attributes: { lesson_toc_target: 'lessonContent' }) do |component| %>
+        <%= @lesson.body.html_safe %>
+
+        <% component.with_footer do %>
+          <div class="pt-10">
+          <%= link_to github_edit_url(@lesson), target: :_blank, rel: 'noreferrer noopener', class: 'underline hover:no-underline text-gray-600 hover:text-gray-800 dark:text-gray-400 dark:hover:text-gray-300 px-5' do %>
+
+          <span class='fab fa-github mr-1'></span>
+
+          <span>
+            Improve on GitHub
+          </span>
+          <% end %>
+
+          <%= link_to github_report_url(@lesson), target: :_blank, rel: 'noreferrer noopener', class: 'underline hover:no-underline text-gray-600 hover:text-gray-800 dark:text-gray-400 dark:hover:text-gray-300 px-5' do %>
+
+          <span class='fas fa-flag mr-1'></span>
+
+          <span>Report an issue/suggestion</span>
```

Comment:
I absolutely get the reasoning behind adding suggestion and I don't mean to be difficult, it just reads a little awkward to me with report and suggestion together - get a second opinion on that lol.

In this case our issue template probably needs updated by dropping "suggestion" from it. Lately, we've been directing people to start a GitHub discussion when they have suggestions of things we could add or ideas of how to improve the curriculum. Our issues should only really be for stuff thats broken and immediately actionable.

One other thought 🤔 we could maybe link to the [choose screen](https://github.com/TheOdinProject/curriculum/issues/new/choose) instead to cover all our bases. And have "Suggest an improvement" as the text. That would pair really well with the "Improve on Github" link.

Reply:
Yeah let's link to the choose screen. That covers the bases with report/suggest going to "Suggest an improvement"